docs/sources/alerting/set-up/configure-roles/index.md

```diff
@@ -0,0 +1,79 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/set-up/configure-roles/
+description: Configure roles and permissions for Grafana Alerting
+keywords:
+  - grafana
+  - alerting
+  - set up
+  - configure
+  - roles and permissions
+labels:
+  products:
+    - oss
+title: Configure roles and permissions
+weight: 150
+refs:
+  org-roles:
+    - pattern: /docs/grafana/
+      destination: https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/roles-and-permissions#organization-roles
+---
+
+# Configure roles and permissions
+
+A user is any individual who can log in to Grafana. Each user is associated with a role that includes permissions. Permissions determine the tasks a user can perform in the system. For example, the Admin role includes permissions for an administrator to create and delete users.
+
+For more information, refer to [Organization roles][ref:org-roles]
+
+## Manage access using roles
+
+For Grafana OSS, there are three roles: Admin, Editor, and Viewer.
+
+Details of the roles and the access they provide for Grafana Alerting are below.
+
+| Role   | Access                                                                                                                                                                    |
+| ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Admin  | Write access to alert rules, notification resources (notification API, contact points, templates, time intervals, notification policies, and silences), and provisioning. |
+| Editor | Write access to alert rules, notification resources (notification API, contact points, templates, time intervals, notification policies, and silences), and provisioning. |
+| Viewer | Read access to alert rules, notification resources (notification API, contact points, templates, time intervals, notification policies, and silences).                    |
+
+## Assign roles
+
+To assign roles, admins need to complete the following steps.
+
+1. Navigate to **Administration** -> **Users and access** -> **Users, Teams, or Service Accounts**.
```

Comment:
Yesterday I found out that we have style guidance to just use the `>` symbol: https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold:~:text=Use%20bold%20to%20indicate%20paths%20within%20a%20web%20application%2C%20and%20greater%2Dthan%20symbols%20(%3E)%20to%20indicate%20path%20separators.%20For%20example%3A

```suggestion
1. Navigate to **Administration** > **Users and access** > **Users, Teams, or Service Accounts**.
```